<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientSpace</class>
 <widget class="QDialog" name="ClientSpace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>326</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>985</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Nome :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_name">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Cart√£o : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_card">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Morada : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_address">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="bot_menu">
         <property name="text">
          <string>Menu</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>Cliente</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Marca</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Modelo</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ano</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Cilindrada</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Potencia</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Matricula</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
